drjt/Animal/lib/LivingCreature.pm

```diff
@@ -41,13 +41,15 @@ if you don't export anything, such as for a purely object-oriented module.
 =cut
 
 sub speak {
-  my $class = shift;
+  my ( $out, $class ) = ( undef, shift );
```

Comment:
This reads a bit strangely, as you're defining a new variable on the same line as retrieving a parameter. I'd do these on separate rows.